backend/src/events/controllers.ts

```diff
@@ -189,6 +189,194 @@ const getPastEvents = async (req: Request, res: Response) => {
   }
 };
 
+/**
+ * Get all event info by eventID
+ * @returns promise with all event info or error
+ */
+const getEvent = async (req: Request, res: Response) => {
+  // #swagger.tags = ['Events']
+  try {
+    const eventID = req.params.eventid;
+
+    const event = await prisma.event.findUnique({
+      where: {
+        id: eventID,
+      }
+    });
+    res.status(200).json(event);
+  } catch (error) {
+    const result = (error as Error).message;
+    res.status(500).json({ result });
+  }
+};
+
+
+/**
+ * Gets all attendees registered for an event
+ * @returns promise with all attendees of the event or error
+ */
+
+const getAttendees = async (req: Request, res: Response) => {
+  //#swagger.tags = ['Events']
+  try {
+    const eventID = req.params.eventid;
+
+    const attendees = await prisma.eventEnrollment.findMany({
+      where: {
+        eventId: eventID,
+      },
+      include:{
+        user: true
+      }
+    })
+    res.status(200).json(attendees);
+  } catch (error: any) {
+    res.status(500).json({ error: error.message });
+  }
+};
+
+
+/**
+ * Adds specified user to an event
+ * @returns promise with user or error
+ */
+
+const addAttendee = async (req: Request, res: Response) => {
+  //#swagger.tags = ['Events']
+ 
+
+  try {
+    const attendee = await prisma.eventEnrollment.create({
+      data: {
+        event: {
+          connect: {
+            id: req.params.eventid,
+          }
+        },
+        user: {
+          connect: {
+            id: req.params.attendeeid,
+          }
+        }
+      }
+    });
+
+    res.status(200).json(attendee);
```

Comment:
Actually returning the `attendee` here is not the right approach because you are returning the entire eventEnrollment table. Look at the prisma docs or other code in the controllers file for how to use the `include` keyword